peers/cache_test.go

```diff
@@ -50,19 +64,19 @@ func TestMultipleTopics(t *testing.T) {
 	require.NoError(t, err)
 	topics := []discv5.Topic{discv5.Topic("first"), discv5.Topic("second")}
 	for i := range topics {
-		peers := [3]*discv5.Node{
-			discv5.NewNode(discv5.NodeID{byte(i), 1}, net.IPv4(100, 100, 0, 3), 32311, 32311),
-			discv5.NewNode(discv5.NodeID{byte(i), 2}, net.IPv4(100, 100, 0, 4), 32311, 32311),
-			discv5.NewNode(discv5.NodeID{byte(i), 3}, net.IPv4(100, 100, 0, 2), 32311, 32311)}
-		for _, peer := range peers {
+		for j := 0; j < 3; j++ {
+			identity, err := crypto.GenerateKey()
+			require.NoError(t, err)
+			peer := discv5.NewNode(
+				discv5.PubkeyID(&identity.PublicKey),
+				net.IPv4(100, 100, 0, byte(i)),
+				32311,
+				32311)
 			assert.NoError(t, peersDB.AddPeer(peer, topics[i]))
 		}
 	}
 	for i := range topics {
 		nodes := peersDB.GetPeersRange(topics[i], 10)
 		assert.Len(t, nodes, 3)
-		for _, n := range nodes {
```

Comment:
Is it useless check?

Reply:
Yes, what we care about is that there are three nodes, nodeID is irrelevant.